VAS.Core/Common/StateController.cs

```
@@ -538,7 +538,9 @@ public override bool Equals (object obj)
 		public async Task<bool> Unload ()
 		{
 			bool result = await ScreenState.UnloadState ();
-			Completion.SetResult (result);
+			if (Completion.Task.Status != TaskStatus.RanToCompletion) {
```

After our chat: Add a `TechnicalDebt` task, and a FIXME for this, as it's hiding an exception